modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -17,25 +18,7 @@ validParams<RadialReturnStressUpdate>()
                              "return the isotropic stress state to a J2 yield surface.  This class "
                              "is intended to be a parent class for classes with specific "
                              "constitutive models.");
-  // Newton Iteration control parameters
-  params.addParam<bool>(
-      "output_iteration_info",
-      false,
-      "Set true to output newton iteration information from the radial return material");
-  params.addParam<bool>(
-      "output_iteration_info_on_error",
-      false,
-      "Set true to output the recompute material iteration information when a step fails");
-  params.addParam<Real>(
-      "relative_tolerance",
-      1e-8,
-      "Relative convergence tolerance for the newton iteration within the radial return material");
-  params.addParam<Real>(
-      "absolute_tolerance",
-      1e-20,
-      "Absolute convergence tolerance for newton iteration within the radial return material");
-  params.addParam<unsigned int>(
-      "max_iterations", 30, "Maximum number of newton iterations in the radial return material");
```

Comment:
We have a bit of a problem here because the new base class takes `max_its`.  We should probably add `max_iterations` to the base class to avoid breaking models.